server/dive_tasks/tasks.py

```diff
@@ -206,7 +207,7 @@ def run_pipeline(self: Task, params: PipelineJob):
 
         # Download source media
         input_folder: GirderModel = gc.getFolder(input_folder_id)
-        input_media_list = utils.download_source_media(gc, input_folder_id, input_path)
+        input_media_list, _ = utils.download_source_media(gc, input_folder_id, input_path)
```

Comment:
I think it is python standard to use `_` for ignored return values.  How do you fell about actually giving it a descriptive name still (just so it involves one less code hop to figure it out) with it being `_input_type`?

Reply:
I think that might cause confusion for readers of this code if they wonder why `_input_type` isn't used anywhere else. IDEs should help out by showing you the the type of `download_source_media` as well. I think it's best to stick with standard practice here.

Reply:
I'm going to resolve this one.